packages/profiling-node/bindings/cpu_profiler.cc

```diff
@@ -111,14 +111,13 @@ class MeasurementsTicker {
 
   size_t listener_count();
 
-  ~MeasurementsTicker() {
+  void Cleanup() {
     uv_timer_stop(&timer);
 
     auto handle = reinterpret_cast<uv_handle_t *>(&timer);
 
-    // Calling uv_close on an inactive handle will cause a segfault.
-    if (uv_is_active(handle)) {
-      uv_close(handle, nullptr);
+    if(!uv_is_closing(handle)) {
```

Comment:
It feels like this should be `uv_is_active() && !uv_is_closing()`

Reply:
uv will only assert that this is closing, so I think we are fine, but good call yes